.github/workflows/kuksa_val_docker.yml

```diff
@@ -5,15 +5,13 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-    paths-ignore:
-    - ".github/workflows/kuksa_databroker_build.yml"
-    - "kuksa_databroker/**"
-    - "kuksa_go_client/**"
-    - "kuksa_client/**"
-    - "Cargo.*"
-    - "Cross.toml"
+    paths:
+    - ".github/workflows/kuksa_val_docker.yml"
+    - "kuksa-val-server/**"
+    - "kuksa_certificates/**"
+    - "proto/**"
```

Comment:
do we need proto here? Server doesn't use it or am I wrong?

Reply:
Correct, good catch, it uses it's own proto file at https://github.com/eclipse/kuksa.val/blob/master/kuksa-val-server/protos/kuksa.proto. Will update